cmd/containerd/main_windows.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"context"
+	"net"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"google.golang.org/grpc"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/docker/containerd/execution"
+	"github.com/urfave/cli"
+)
+
+func appendPlatformFlags(flags []cli.Flag) []cli.Flag {
+	return flags
+}
+
+func processRuntime(ctx context.Context, runtime string, root string) (execution.Executor, error) {
+	// TODO
+	return nil, nil
```

Comment:
Returning `nil, nil` has the function returning success with nothing there. These need to break.